modules/core/shared/src/main/scala/lucuma/core/model/Tracking.scala

```diff
@@ -3,11 +3,24 @@
 
 package lucuma.core.model
 
-import cats._
-import lucuma.core.math._
+import cats.Eq
+import cats.Show
+import lucuma.core.math.*
+import lucuma.core.util.Timestamp
 import monocle.Focus
 import monocle.Lens
 
+/** A coordinate along with a rate of change in RA and Dec for some time unit
+  *
+  * @param coord coordinates
+  * @param delta rate of change in RA and dec, where the delta(RA)/time has been
+  *              multiplied by the cosine of the dec
+  */
+sealed trait TrackedCoordinates {
+  def coord: Coordinates
+  def delta: Offset /* per time unit */
```

Comment:
Would we rather express this as `AngularVelocity` to make it clear the time component?

Reply:
It seems like, for `delta` to be generically useful regardless of target type, you'd have to know the time unit in question.

Reply:
@tpolecat I think you're the expert in Horizons, would you shed some light about the delta component? Is it required for the TCC?

Reply:
I don't have any specific insight here. Horizons returns velocities in arcsrc/hr IIRC so we're storing them in OCS, but I don't know if they're used for guiding. We're doing our own interpolation and I don't know if these values are used at all.

Reply:
But in ay case yes, if we're storing this it should be a velocity rather than a scalar.

Reply:
If I recall correctly, we put them into the ephemerides files and the TCS uses them.

Reply:
Yes, the TCS expects to see it at least but I don't know whether it takes it into account.

https://github.com/gemini-hlsw/ocs/blob/develop/bundle/edu.gemini.spdb.reports.collection/src/main/scala/edu/gemini/dbTools/ephemeris/EphemerisFileFormat.scala#L65-L67